src/state_point.cpp

```diff
@@ -421,9 +430,10 @@ void load_state_point()
   read_dataset(file_id, "current_batch", simulation::restart_batch);
 
   if (simulation::restart_batch >= settings::n_max_batches) {
-    fatal_error(fmt::format(
-      "The number of batches specified for simulation ({}) is smaller"
-      " than the number of batches in the restart statepoint file ({})",
+    warning(fmt::format(
```

Comment:
With that `if` statement you added earlier, doesn't that mean we will just show this warning then exit? Just making sure I understand correctly.

Reply:
That's correct, but with this change OpenMC will no longer won't exit here. When using OpenMC as a library, this is important b/c a statepoint file (and its results) can be loaded and accessed without killing the library even if the number of batches in the file relative to those set in the simulation isn't appropriate for a restart run.